Runtime/BacktraceClient.cs

```diff
@@ -28,6 +28,8 @@ public class BacktraceClient : MonoBehaviour, IBacktraceClient
         /// </summary>
         private readonly Dictionary<string, string> _clientAttributes = new Dictionary<string, string>();
 
+        public List<string> Attachments { get; internal set; } = new List<string>();
```

Comment:
Generic across the file: I don't like the name `Attachments`. It's not what's in the collection. It's paths to attachments file to upload.